### ISSUE TITLE
Provide real installation instructions

### ISSUE THREAD

**Issue by fresheneesz** [flot/flot#1394]
Saying "Just include the Javascript file after you've included jQuery" is incredibly  unhelpful. WHICH javascript file? Please be more detailed.

**Comment by rhandberg** [comment]
I completely agree with this issue!!!
This is, to say the least, wildly unhelpful! Which JavaScript file? Should I include all the plugins? If not, which ones? Are there minified versions somewhere?

And saying

> They've been successfully been used by thousands of developers before...

is simply condescending to new users wanting to use Flot...

**Comment by andig** [comment]
> is simply condescending to new users wanting to use Flot...

I agree. While my answer was in the tone of the original post, I shouldn‘t have put it like that and apologize. I still think that the examples are helpful.
